addons/beehave/nodes/beehave_tree.gd

```diff
@@ -93,3 +100,22 @@ func enable() -> void:
 
 func disable() -> void:
 	self.enabled = false
+
+
+func _enter_tree() -> void:
+	# Get the name of the parent node name for metric
+	var parent_name = get_parent().name
+	_tree_metric_name = "beehave/process_time/%s(%s)" % [parent_name, get_instance_id()]
```

Comment:
From a structural perspective, we may want to have multiple metrics as part of the same tree. Therefore, I'd group them differently:
```suggestion
	_tree_metric_name = "beehave/%s-%s/process_time" % [parent_name, get_instance_id()]
```